circle.yml

```diff
@@ -17,6 +17,7 @@ deployment:
       - chmod 600 ~/.netrc
       - "[[ ! -s \"$(git rev-parse --git-dir)/shallow\" ]] || git fetch --unshallow"
       - DEPLOY_ENV=staging yarn deploy
+      - git push git@github.com:artsy/force.git master:staging
```

Comment:
I think this might want to push `ENV['CIRCLE_SHA1']` instead of `master`. I think the local `master` branch _should_ point to the same commit but this is more explicit anyway (and works even if other deploy commands end up refreshing local `master` from the remote).